components/end/end.app.mjs

```diff
@@ -1,11 +1,13 @@
 export default {
   type: "app",
   app: "end",
-  propDefinitions: {},
-  methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+  propDefinitions: {
+    reason: {
+      type: "string",
+      label: "Reason",
+      description: "A reason why the workflow execution was ended",
```

Comment:
nitpicking, but this wording is awkward to. does it need to be in passive voice? maybe something like, "Reason for terminating / exiting / ending workflow execution", for example